src/Tabs/Tabs.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Flex from '../Flex';
+import Tab from './Tab';
+import { ENTER_KEY, SPACEBAR } from '../constants';
+
+class Tabs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const { activeTabId, children } = this.props;
+
+    this.state = {
+      activeTabId: activeTabId || children[0].props.id
+    };
+  }
+
+  get activeTabContent() {
+    const { activeTabId } = this.state;
+    const { children } = this.props;
+    const activeTab = children.find(({ props: { id } }) => id === activeTabId);
+
+    return activeTab.props.content;
+  }
+
+  activateTab = tab => {
+    const { id } = tab.props;
+
+    this.setState({ activeTabId: id });
+  };
+
+  handleKeyPress = (key, tab) => {
+    if (key === ENTER_KEY || key === SPACEBAR) {
+      this.activateTab(tab);
+    }
+  };
+
+  render() {
+    const { activeTabId } = this.state;
+    const { Component, children, ...props } = this.props;
+
+    return (
+      <>
+        <Component {...props}>
+          {children.map(tab => {
+            const { content, id } = tab.props;
+            const active = activeTabId === id;
+            const contentId = content.props.id || `${id}-tab-content`;
+
+            return React.cloneElement(tab, {
+              'aria-controls': contentId,
+              'aria-selected': active ? 'true' : 'false',
+              active,
+              content: React.cloneElement(content, { id: contentId }),
```

Comment:
This could potentially be simplified if we go the route of passing the content in as children instead as noted below.

Instead of having to worry about mutating the id prop here, we could just wrap the children in a new container with the id instead. That way it doesn't matter if the user passed in a component with an id already.

Reply:
This piece is probably the most convincing argument to make the content the child. What are your thoughts with the feedback above?

Reply:
I think I'm OK with taking the children prop out of the tab and using it alongside of the tabs themselves.